forge-core/src/main/java/forge/card/mana/ManaCost.java

```diff
@@ -281,6 +281,9 @@ public String toString() {
      * @return
      */
     public String getShortString() {
+        if (isNoCost()) {
+            return "-1";
```

Comment:
I don't like that one

I would have preferred the "no cost" stuff

Reply:
Hmn, but this is basically the definition for it:
https://github.com/Card-Forge/forge/blob/42893234b36f5d889702d4c55ddf4afb20ccb98f/forge-core/src/main/java/forge/card/mana/ManaCost.java#L50-L50

If I'd write it out it would be treated as two different parts and fail the parsing 🤷‍♂️

Reply:
ah there is the no cost i was looking for:
```java
face.setManaCost("no cost".equals(value) ? ManaCost.NO_COST
                                : new ManaCost(new ManaCostParser(value)));
```

it is only used for reading cardfaces

Reply:
while i'm not happy about that "-1", that can be fixed later